### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by lbeeon**
It takes me a little of time to find the end of pipe trigger event.
The event 'end' in through2.obj is not work with base use case, to catch the callback of the base use case should listen on 'finish' event.

### CODE REVIEW

**Comment by Delapouite** [comment]
Thanks! You made a small typo:  doSomthingSpecial()

**Comment by lbeeon** [comment]
I fixed the typo, thanks.

File: README.md [hunk 1]
```diff
@@ -20,6 +20,9 @@ fs.createReadStream('ex.txt')
     callback()
    }))
   .pipe(fs.createWriteStream('out.txt'))
+  .on('finish', function (){
```
Comment (rvagg) [inline README.md]: tiny nit, sorry, can you put a space before `{` please?

**Comment by rvagg** [comment]
thanks @lbeeon, landed @ c523347

**Pull request closed without merging**